sdks/java/core/src/main/java/org/apache/beam/sdk/util/WindowedValue.java

```diff
@@ -716,25 +714,24 @@ public CloudObject initializeCloudObject() {
   /**
    * Coder for {@code WindowedValue}.
    *
-   * <p>A {@code ValueOnlyWindowedValueCoder} only encodes and decodes the value. It drops
-   * timestamp and windows for encoding, and uses defaults timestamp, and windows for decoding.
+   * <p>A {@code ValueOnlyWindowedValueCoder} only encodes and decodes the value. It drops timestamp
+   * and windows for encoding, and uses defaults timestamp, and windows for decoding.
```

Comment:
Fix some minor grammatical issues:
It drops timestamp and windows when encoding, and uses a default timestamp and window when decoding.